pkg/cmd/roachtest/roachtestutil/mixedversion/planner.go

```diff
@@ -1394,6 +1416,19 @@ func (ss stepSelector) CutBefore(predicate func(*singleStep) bool) (stepSelector
 	return append(before, cut...), after
 }
 
+// MarkNodesUnavailable marks all steps in the given step selector
+// as being in the context of a failure injection.
```

Comment:
nit: lets explain what "in the context of a failure injection" means here. Is it that system nodes might be unavailable? Why does that matter?